### PR TITLE
This is the new title

### PR DESCRIPTION
**Pull request by SebastianO1717**

### CODE REVIEW

File: Extensions/Program.cs [hunk 1]
```diff
@@ -1,2 +1,46 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿/// <summary>
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+///
```
Comment (SebastianO1717) [inline Extensions/Program.cs:7]: can we remove this spacing?

Comment (Sebastian1712) [inline Extensions/Program.cs:7]: Please remove spacing

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: I'll go ahead and fix this

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: hello :)

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: Wow

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: this is working just fine

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: Not fine

**Pull request closed without merging**